create/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script defer src="https://cdn.jsdelivr.net/npm/@supabase/supabase-js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../styles/reset.css">
+    <link rel="stylesheet" href="../styles/style.css">
+    <link rel="stylesheet" href="./create.css">
+    <title>Create</title>
+</head>
+<body>
+   <header>
+
+   </header> 
+   <main>
+    <a href="/" id="home">Home</a>
+    <div class="container">
+        <h1>Workshop Sign up form</h1>   
+        <form id="form">
+                <label for="input">Participant Name
+                    <input type="text" name="input" required>
+                </label>
+                <label for="select">Workshop
+                    <select name="select" id="select">
+                        <option selected>Please select an option..</option>
```

Comment:
Really nice detail to add, makes sure the user properly selects a workshop. nice job!